components_new/asp/asp.py

```
@@ -0,0 +1,85 @@
+        with DBClient() as db:
+            self.db = db
+            self.eventWriter = EventWriter(db)
+            self.sysWriter = SystemWriter(db)
+            self.monitorFolder()
+
+
+    def monitorFolder(self):
+        '''
+            Parse the system log files contained in folder.
+        '''
+        visitedFiles = set()
```

Ruff (0.8.2)

Local variable `visitedFiles` is assigned to but never used

Remove assignment to unused variable `visitedFiles`

(F841)